src/commands/Dev/test/loopBackAddress.test.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, expect, test } from '@jest/globals';
```

Comment:
Is this test case for localhost option?
If so, I think it's better with localhost.test.ts file name.

Reply:
I changed to localhost.test:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/761eb859467b4e3dce2a64666dd4ad1c98a47246

Reply:
Confirmed.